nautobot/core/templates/inc/nav_menu.html

```diff
@@ -2,27 +2,57 @@
 {% load registry %}
 {% load helpers %}
 {% registry %}
-<nav class="navbar navbar-default navbar-fixed-top">
+<nav class="navbar navbar-default navbar-fixed-left navbar-inverse">
     <div class="container-fluid">
         <div class="navbar-header">
-            <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false">
-                <span class="sr-only">Toggle navigation</span>
-                <span class="icon-bar"></span>
-                <span class="icon-bar"></span>
-                <span class="icon-bar"></span>
-            </button>
             <a class="navbar-brand" href="{% url 'home' %}">
                 <img src="{% custom_branding_or_static 'logo' 'img/nautobot_logo.svg' %}" height="30" />
             </a>
         </div>
 
-        <div id="navbar" class="navbar-collapse collapse">
+        <div id="navbar">
+            {% if request.user.is_authenticated or not "HIDE_RESTRICTED_UI"|settings_or_config %}
+            <form action="{% url 'search' %}" method="get" class="navbar-form" id="navbar_search" role="search">
+                <div class="input-group">
+                    <input type="text" name="q" class="form-control" placeholder="Search {{ settings.BRANDING_TITLE }}">
+                    <span class="input-group-btn">
+                            <button type="submit" class="btn btn-primary">
+                                <i class="mdi mdi-magnify"></i>
+                            </button>
+                        </span>
```

Comment:
Indentation looks off here.

```suggestion
                    <span class="input-group-btn">
                        <button type="submit" class="btn btn-primary">
                            <i class="mdi mdi-magnify"></i>
                        </button>
                    </span>
```

Reply:
It sure is!